spec/spec_helper_acceptance.rb

```diff
@@ -22,14 +22,33 @@
   c.before :suite do
     # Install module and dependencies
     hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
+      if fact_on(host, 'os.family') == 'RedHat'
         # don't delete downloaded rpm for use with BEAKER_provision=no +
         # BEAKER_destroy=no
         on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
         # refresh check if cache needs refresh on next yum command
         on host, 'yum clean expire-cache'
         # We always need EPEL
         on host, puppet('resource', 'package', 'epel-release', 'ensure=installed')
+
+        major = fact_on(host, 'os.release.major')
+
+        if major == '8'
+          on host, 'dnf -y module enable pki-core'
+          on host, 'dnf -y install glibc-langpack-en'
```

Comment:
I think this is preferred way to do package installs: https://github.com/theforeman/puppet-candlepin/pull/148/files#diff-174f7dfb508155027cb4c3e9a1840591R32

This can be applied to line 50 as well for the candlepin-selinux RPM.

Reply:
I indeed have preferred `host.install_package('glibc-langpack-en')` because it's a bit more explicit about what you want to achieve and less about the how.